### PR TITLE
Now watch and learn, here's the deal, don't slip and slide on this banana peel!

### PR DESCRIPTION
**Pull request by boy2mantwicethefam**
Banana peels can now slip you a few spaces forward, it's a dice roll between slipping only on the spot and slipping up to 3 spaces away, and the banana peel follows you too
Bluespace peels and syndicate peels are not affected
RIP Stefan
:cl:
 * tweak: Banana peels are now more slippery, meaning you can slide a few tiles forward if you walk into one.

### CODE REVIEW

**Comment by calzilla1** [comment]
If this goes through, then there should be a syndicate variant that always slips a bunch of tiles forward (think lube slips), stays in place, and automatically opens any doors the slipee goes through without stopping.

**Comment by boy2mantwicethefam** [comment]
The traitor banana peel already does that minus the door opening part

**Comment by calzilla1** [comment]
> The traitor banana peel already does that minus the door opening part

There's not a singular peel in the uplink, but there is the banana grenade which is more for large scale chaos or escape. The one I'm talking about is more for utility/ assassination (slipping people out airlocks, slipping people into the SM, slipping yourself into restricted areas, etc)

**Comment by Kurfursten** [comment]
The only issue with this seems to be that it carries over the functionality to clownfish. Intended?

File: code/game/objects/items/weapons/clown_items.dm [hunk 1]
```diff
@@ -11,10 +11,23 @@
...
+		if(slip_n_slide(M))
 			M.simple_message("<span class='notice'>You slipped on the [name]!</span>",
 				"<span class='userdanger'>Something is scratching at your feet! Oh god!</span>")
 
+/obj/item/weapon/bananapeel/proc/slip_n_slide(var/mob/living/carbon/M)
+	if(!M.Slip(2,2,1))
+		return 0
+	var/tiles_to_slip = rand(0,3)
+	if(tiles_to_slip && !locked_to) //The banana peel will not be dragged along so stop the ride
+		M.lock_atom(src)
```
Comment (Kurfursten) [inline code/game/objects/items/weapons/clown_items.dm]: this needs a locking category

**Comment by KingofHONK** [comment]
It's already perfect, but make the chance to go further tiles based on potency and it's perfect.

**Comment by SonixApache** [comment]
fix it

**Comment by boy2mantwicethefam** [comment]
next pr @KingofHONK

**Pull request merged**